toolbox/library/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ list(APPEND WBT_BLOCKS "Jacobian")
 list(APPEND WBT_BLOCKS "ForwardKinematics")
 list(APPEND WBT_BLOCKS "RelativeTransform")
 list(APPEND WBT_BLOCKS "CentroidalMomentum")
+list(APPEND WBT_BLOCKS "CMM")
```

Comment:
I would extend CMM to CentroidalTotalMomentumMatrix pretty much everywhere (class names, files names, etc)

Reply:
Perfect, I will apply this change right away!

Reply:
Done in commit 95fa7bcb51c3f942271b80777c4861f7eb814e3c